packages/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,38 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+const getOrderItemsByUserId = async (token) => {
+  try {
+    if (!token) {
+      throw new HttpError('You are not authenticated', 401);
+    }
+
+    const userUid = token.split(' ')[1];
+
+    if (!userUid) {
+      throw new HttpError('Token is wrong', 400);
+    }
```

Comment:
I don't think this code makes much sense in this kind of non-production app. 

While it makes a lot of sense to do server side verification, it only makes sense if you're actually doing that and not just doing code that pretends to do that. 
That said, from a learning perspective, I would look into creating an Express middleware to handle this verification instead so that the feature controller don't have to care, they can assume the request is verified already. 
That way you don't need to duplicate this check in every endpoint.